GenericTrigger.lua

```diff
@@ -63,7 +63,7 @@ local tostring, select, pairs, type = tostring, select, pairs, type
 local error, setmetatable = error, setmetatable
 
 -- WoW APIs
-local IsPlayerMoving = IsPlayerMoving
+local apiWrapper = WowApiWrapper:create()
```

Comment:
This kind of change is in a lot of the files.

I could have done the assignment like:
```
local apiWrapper = WowApiWrapper:create()
local IsPlayerMoving = apiWrapper.IsPlayerMoving
```

But that makes it more difficult to find usages of the WoW API.

If everything went through the wrapper then you could fairly easily do a "Find All" for "apiWrapper" to find every access to the WoW API throughout the whole project.